### PR TITLE
Running tests in node

### PR DESCRIPTION
**Pull request by CheezItMan**
One small lesson on running tests in node.  This could be an optional lesson.

### CODE REVIEW

**Review by jericahuang — APPROVED** [review]
👍🏻

File: tests/testing-in-node.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+```bash
+$ yarn install
+```
+
+### Inspect the `test/fizzbuzz.test.js` file
+
+Open the `test/fizzbuzz.test.js` file.  This is a simple test file that tests the `fizzBuzz` function.  
+
+<details>
+  <summary>How does the test import the fizzBuzz function?</summary>
```
Comment (kaidamasaki) [inline tests/testing-in-node.md:44]:
```suggestion
  <summary>How does the test import the `fizzBuzz` function?</summary>
```

File: tests/testing-in-node.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+To run the tests in the terminal we can use following command:
+
+```bash
+$ yarn test
+```
+
+We should get the following output:
+
+```bash
```
Comment (kaidamasaki) [inline tests/testing-in-node.md:62]: This shouldn't be highlighted as bash, it's just output.  (Trying to suggest changes here screws up the suggestion block though.  XD)

File: tests/testing-in-node.md (hunk 1 above)
Comment (CheezItMan) [reply tests/testing-in-node.md:44]: I didn't do this because Learn doesn't handle the backticks well inside a `<details>` block.

**Pull request merged**